### DISCUSSION TITLE
I can't get "multiLinePasteWarning" and "largePasteWarning" set to false

### DISCUSSION THREAD

**Discussion by earcuri** [microsoft/WSL#6049] [Q&A] [answered]
I have set these under the defaults heading but cannot seem to get either of them to work.  
```
"profiles":
    {
        "defaults":
        {
            // Put settings here that you want to apply to all profiles
            "fontFace": "Hack",
            "fontSize": 10,
            "multiLinePasteWarning": false,
            "largePasteWarning": false
        },
```
Is this where they are supposed to go?  These dialog boxes are horrendous for productivity and annoying AF.

TIA.

**Comment by craigloewen-msft** [comment, marked as answer]
I believe you're talking about Windows Terminals settings here right?

If yes, then I believe your question will be better answered at the [Windows Terminal Github repo](http://github.com/microsoft/terminal)!

**Reply by earcuri** [reply]
You are correct.  I will take it over there.  Thanks!!
